dataactcore/models/jobTrackerInterface.py

```diff
@@ -263,37 +263,45 @@ def setJobRowcounts(self, jobId, numRows, numValidRows):
             {"number_of_rows_valid": numValidRows, "number_of_rows": numRows})
         self.session.commit()
 
-    def getSubmissionStatus(self,submission_id):
-        job_ids = self.getJobsBySubmission(submission_id)
+    def getSubmissionStatus(self,submission):
+        job_ids = self.getJobsBySubmission(submission.submission_id)
         status_names = self.getJobStatusNames()
         statuses = dict(zip(status_names,[0]*len(status_names)))
         skip_count = 0
 
         for job_id in job_ids:
             job = self.getJobById(job_id)
-            if job.job_type.name != "external_validation":
+            if job.job_type.name not in ["external_validation", None]:
                 job_status = job.job_status.name
                 statuses[job_status] += 1
             else:
                 skip_count += 1
```

Comment:
Significantly outside the scope for this PR, but we've got an n+1 query situation here (we're hitting the db for each job). It'd probably be worth refactoring to do most of this logic in the db.